### ISSUE TITLE
factions lack accessibility to diplomats 

### ISSUE THREAD

**Issue by asher-k** [VeryEager/DynDaC#7]
Not all factions start with diplomats and thus may not be able to obtain them in a reasonable timeframe, limiting campaign strategy. Factions without starting diplomats should have them added automatically.
